src/Api/Controllers/OrganizationsController.cs

```diff
@@ -333,7 +348,7 @@ public async Task<PaymentResponseModel> PostSeat(string id, [FromBody] Organizat
             throw new NotFoundException();
         }
 
-        var result = await _organizationService.AdjustSeatsAsync(orgIdGuid, model.SeatAdjustment.Value);
+        var result = await _organizationService.AdjustSeatsAsync(orgIdGuid, model.SeatAdjustment.Value, null);
```

Comment:
This change is unnecessary, please revert.

Reply:
yeah, you are right